access-control-app/src/cmr/access_control/api/routes.clj

```diff
@@ -18,19 +17,14 @@
    [cmr.common-app.api.launchpad-token-validation :as lt-validation]
    [cmr.common-app.api.routes :as common-routes]
    [cmr.common-app.services.search.parameter-validation :as cpv]
-   [cmr.common.api.errors :as api-errors]
    [cmr.common.cache :as cache]
-   [cmr.common.log :refer (debug info warn error)]
+   [cmr.common.log :refer (info error)]
    [cmr.common.mime-types :as mt]
    [cmr.common.services.errors :as errors]
    [cmr.common.util :as util]
-   [cmr.common.validations.core :as validation]
-   [compojure.core :refer :all]
-   [compojure.handler :as handler]
-   [ring.middleware.keyword-params :as keyword-params]
-   [ring.middleware.nested-params :as nested-params]
-   [ring.middleware.params :as params])
+   [compojure.core :refer [DELETE GET OPTIONS POST PUT context routes]])
   (:import
+   #_{:clj-kondo/ignore [:unused-import]}
    (org.json JSONException)))
```

Comment:
If this is an unused import, do we really need it to remain here?

Reply:
The import is here so that we don't get a class not found exception when throwing an exception from the access control application. While this namespace may not use it, something down the line does, but the compiler doesn't realize it and so it pulls it out.  We need the import or use statement to make sure the class doesn't get pulled out.